azure-pipelines.yml

```diff
@@ -17,6 +17,10 @@ jobs:
 - job: 'Test'
   variables:
     NOTEBOOK_KERNEL: "thinc-notebook-tests"
+    ${{ if eq(variables['python.version'], '3.6') }}:
+      PYTORCH_VERSION: "1.5.0"
+    ${{ if ne(variables['python.version'], '3.6') }}:
+      PYTORCH_VERSION: "1.11.0"
```

Comment:
I'm not sure `python.version` is defined at this point?. (But I have never understood all the variable substitution details, it's all trial and error.)